client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -162,10 +162,11 @@ class WpcomChecklistComponent extends PureComponent {
 		} );
 	};
 
-	onCollapsedClick = selectedTaskId => {
+	onTaskTitleClick = selectedTaskId => {
```

Comment:
This isn't only called when clicking the task's title, but also the chevron. Should we thus rename it to `onTaskClick`?

(We tried to implement a convention to actually make these action handler function names descriptive wrt what they do as opposed to when they're called -- in this case, we could call it e.g. `expandTask`.)

Reply:
Also, this is generic enough to the checklist UI component that it should probably be moved into `components/checklist` (and removed here and from the JP checklist).